helpers/calculateCabbages.js

```diff
@@ -4,15 +4,19 @@ function getCabbageBreakdown(memberData, playerDetails) {
     // We return an object with key-value pairs (<achievement-name>, <amount of cabbages>)
     // If playerDetails is undefined or not provided, we reverse engineer the core cabbages
     const { accountProgression: account } = memberData;
+    let clogCabbages = Math.floor(account.clogSlots / 100) * 25;
+    Configuration.clogCabbages.forEach(({ milestone, bonusCabbages }) => {
```

Comment:
I would slightly rewrite this to make it more robust, in case the entire value does not exist in the config.json:
```(Configuration.clogCabbages || []).forEach(...)```

Reply:
I changed it in a152219, is that good?